cruise-control/src/yaml/endpoints/rebalance.yaml

```diff
@@ -135,7 +139,7 @@ RebalanceEndpoint:
         description: Upper bound on the bandwidth used to move replicas
         schema:
           type: integer
-          format: int32
+          format: int64
```

Comment:
Do we have a sanity check for the incorrect type definition? If not can we add one?

Reply:
Currently we do not check for parameters value type since we do not have metadata in Request Java class to infer the value type, we are currently checked against [static parameter map](https://github.com/linkedin/cruise-control/blob/b31598a5b57f0f9d8123a35e4b4a0173d42416fc/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/RebalanceParameters.java#L52)
I prefer solve this in the in the following auto-gen patch.